### ISSUE TITLE
[XCUITests] Enable Sync UI Tests after iOS 13.3 update

### ISSUE THREAD

**Issue by isabelrios** [mozilla-mobile/firefox-ios#5958]
After iOS 13.3 update it was not possible to tap or find any element in the fxa website, or even on its navigation bar.
Tests are failing trying to tap on `Close` button, verifying that the `Email textField` exists, typing in it, or trying to tap on `Continue` button. 
<img width="150" alt="Screenshot 2019-12-27 at 16 44 09" src="https://user-images.githubusercontent.com/1897507/71523181-366e7400-28c8-11ea-90e8-0650751a59be.png">

We need to find if there is a workaround to have them working or find a fix so that these tests are back up and running.

For more info see: #5937

@garvankeeley just wondering if you know about something different with this site than others, this issue is only affecting this place. It is not a matter of waiting, you can wait forever once the site is loaded but the elements are not accessible :(

**Comment by isabelrios** [comment]
It is not about the site but the element we are trying to find, looks like it is a general issue affecting iOS 13.3 : https://stackoverflow.com/questions/59326816/why-does-xctest-code-fail-to-find-textfield-by-value-in-wkwebview-in-ios13-3-wo

Not sure there is something we can do about this...

**Comment by isabelrios** [comment]
There are many issues related to this problem already logged in apple, see for example:
https://forums.developer.apple.com/thread/127132

I also open a feedback assistant ticket: https://feedbackassistant.apple.com/feedback/7529542

**Comment by isabelrios** [comment]
Looks like there is new info about this  in apple developer portal: 
```
I received this update this morning: "Potential fix identified - For a future OS update". So, look out for a new beta release I guess?
```

Hopefully this will be fixed in future releases

**Comment by isabelrios** [comment]
Removing from Backlog since there is nothing we can do until Apple releases the fix

**Comment by isabelrios** [comment]
Looks like this issue is fixed on iOS 13.4. Creating PR to enable the tests again
